scripts/runFullTests.fish

```diff
@@ -217,6 +218,7 @@ set CT "$CT""250,runClusterTest1 hot_backup - --dumpAgencyOnError true\n"
 set CT "$CT""250,runClusterTest1 arangobench - --dumpAgencyOnError true\n"
 set CT "$CT""9600,runClusterTest1 chaos - --dumpAgencyOnError true --skipNightly false\n"
 set ST "$ST""250,runSingleTest1 replication2_client -\n"
+set ST "$ST""250,runSingleTest1 replication2_server -\n"
```

Comment:
Do we want to add the `--dumpAgencyOnError true` thing here as we did in https://github.com/arangodb/oskar/pull/370/files#diff-f90dcac39955f6b516159354119dfeaa12aa3b7f1751b6f32fc9fe6832c7ecd3R183?
Not sure if it is necessary, but it looks a bit inconsistent right now.

Reply:
```suggestion
set ST "$ST""250,runSingleTest1 replication2_server - --dumpAgencyOnError true\n"
```